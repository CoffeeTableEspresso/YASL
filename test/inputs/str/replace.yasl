echo 'YASL'->replace('T', 'S')
echo 'YASL'->replace('SL', '')
