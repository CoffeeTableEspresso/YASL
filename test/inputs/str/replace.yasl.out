YASL
YA
AAA
BAA
BBA
BBB
BBB
